src/CardView/styles.js

```diff
@@ -0,0 +1,68 @@
+import {
+  StyleSheet,
+} from 'react-native'
+
+
+const styles = StyleSheet.create({
+  cardContainer: {},
+  cardFace: {},
+  icon: {
+    position: 'absolute',
+    top: 15,
+    right: 15,
+    width: 60,
+    height: 40,
+  },
+  resizeMode: 'contain',
```

Comment:
I think the `resizeMode` must be inside the icon obj.

Reply:
I didn't worked on style refac yet, this one is legacy.